README.md

```diff
@@ -88,6 +89,7 @@ Name                                                      | Status       | Short
 [mir-opt-stream]: https://rust-lang.zulipchat.com/#narrow/stream/189540-t-compiler.2Fwg-mir-opt
 [pipelining-stream]: https://rust-lang.zulipchat.com/#narrow/stream/195180-t-compiler.2Fwg-pipelining
 [polonius_stream]: https://rust-lang.zulipchat.com/#narrow/stream/186049-t-compiler.2Fwg-polonius
+[learning_stream]: https://rust-lang.zulipchat.com/#narrow/stream/<wg-learning-stream>
```

Comment:
We need to open a stream, I don't have rights to do so.

Reply:
https://rust-lang.zulipchat.com/#narrow/stream/196385-t-compiler.2Fwg-learning

Reply:
```suggestion
[learning_stream]: https://rust-lang.zulipchat.com/#narrow/stream/196385-t-compiler.2Fwg-learning
```

Reply:
Done